contracts/vesting/src/contract.rs

```diff
@@ -189,10 +189,12 @@ impl VestingTrait for Vesting {
             save_vesting(
                 &env,
                 &vesting_schedule.recipient.clone(),
-                &VestingInfo {
+                VestingInfo {
                     balance: vested_amount,
                     recipient: vesting_schedule.recipient,
                     schedule: vesting_schedule.curve.clone(),
+                    index: 0, // just used as a placeholder, `save_vesting` will overwrite this
+                              // value
```

Comment:
Come on, no. Just create another structure `VestingInfoResponse` and include the `index` field there. 
Don't do some hacky shenenigans like this, why risking breaking something?

Reply:
fixed now.

yeah, I knew it was hacky when I was writing the code, but I don't expect it to fail. Shouldn't have even be trying to push that in the first place. Made the new response struct with more clean approach